src/middleware/actionLogger/actionLogger.spec.js

```diff
@@ -0,0 +1,13 @@
+import { setupStore } from '../../testUtils/setupStore';
+
+describe('redux middleware::actionLogger', () => {
+  test('it logs actions', () => {
```

Comment:
Should this be `test()` instead of `it()` (like all other tests)?

Reply:
I believe it's an alias and they're interchangeable, but its better to be consistent.